codeserver/c9s-python-3.9/nginx/serverconf/proxy.conf.template

```diff
@@ -1,13 +1,13 @@
 ###############
-# api calls from probes get to VSCode /healthz endpoint
+# api calls from probes get to Code-Server /healthz endpoint
```

Comment:
```suggestion
# api calls from probes get to Code Server /healthz endpoint
```

Code Server docs seem to either say "Code Server", capitalized, with space, or "code-server", lowercase and with hyphen (same as the cli command is named). I think we should not create our own hybrid.

Reply:
Shall i mark this as resolved ? @jiridanek

Reply:
So, you want to keep writing Code-Server? Which is a form that nobody else is using? Up to you, it was just a suggestion.

Reply:
is the suggestion to keep using lower-case `code-server` ? i got little confused with your first two comments.

Reply:
Well, Jiri is right. Based on the https://github.com/coder/code-server/tree/main/docs, they always use `code-server`. So if that makes sense for us, we shall follow that convention, I think.

Reply:
that's because I was trying to suggest things while I was myself confused

Yes, suggestion is to go for lower case hyphenated `code-server`, which is what Coder uses in their writing, eg. https://github.com/coder/code-server/blob/main/docs/FAQ.md and https://coder.com/docs/code-server/latest/install

Code Server is Microsoft thing, https://code.visualstudio.com/docs/remote/vscode-server. So if we wrote it that way, we might touch their trademark; so better stay away, I think.

Reply:
Yes we will be making change everywhere to stay align with lowercase `code-server`.

Reply:
Thanks @harshad16 - will update docs to align with this.

Reply:
Marking this one as Resolved.